src/main/java/com/jenkinsci/plugins/badge/dsl/AbstractAddBadgeStep.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2010, Sun Microsystems, Inc., Serban Iordache

```

Comment:
I'm not sure that you want to assign copyright for this file to Serban Iordache.  I think that you can either remove the explicit copyright statement or add your additional copyright if the original source was derived from something that Serban Iordache created.
```suggestion
 * Copyright (c) 2004-2010, Sun Microsystems, Inc., Serban Iordache
```